test/test-framework/pkg/apis/cache/v1alpha1/dummy_types.go

```diff
@@ -77,13 +77,13 @@ type Hog struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors=true
 	Inlined InlinedComponent `json:",inline"`
 	// Fields should be inlined
-	InlinedComponent
+	InlinedComponent `json:",inline"`
```

Comment:
Are the `json` tag changes in this file related to this PR?

Reply:
Yes, the CRD generator seems to complain if no JSON tags are present on a field.